### PR TITLE
Add a ConversationError component for rendering unboxing errors

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers 

Screenshots:

screen shot 2017-07-17 at 10 15 57 pm

screen shot 2017-07-17 at 10 18 42 pm

### CODE REVIEW

File: shared/chat/conversation/index.desktop.js [hunk 1]
```diff
@@ -107,6 +108,10 @@ class Conversation extends Component<void, Props, State> {
         </Text>
       </Box>
 
+    if (this.props.conversationIsError) {
```
Comment (chrisnojima) [inline shared/chat/conversation/index.desktop.js]: we shouldn't have these types of switching in the components. instead you should add a branch in the container

Comment (chrisnojima) [inline shared/chat/conversation/index.desktop.js]: see https://github.com/keybase/client/pull/7799/files#diff-33a5dd5b381317dcabdfc216839778efR136

File: shared/chat/conversation/index.native.js [hunk 2]
```diff
@@ -55,25 +56,27 @@ const Conversation = (props: Props) => (
...
-                onScrollDown={props.onScrollDown}
-                onFocusInput={props.onFocusInput}
-                editLastMessageCounter={props.editLastMessageCounter}
-              />
-              <Banner />
-              {props.showLoader && <LoadingLine />}
-              {props.finalizeInfo
-                ? <OldProfileResetNotice />
-                : <Input
+          : props.conversationIsError
```
Comment (chrisnojima) [inline shared/chat/conversation/index.native.js]: same here. this existing code should also change but we can do that in a different ticket

**Comment by cjb** [comment]
@chrisnojima Thanks! Way cleaner now. Filed https://keybase.atlassian.net/browse/DESKTOP-4387 for cleanup.

**Comment by cjb** [comment]
@chrisnojima (bump)

**Pull request merged**